src/index.html

```diff
@@ -52,55 +32,9 @@
       </section>
       <section class="products">
         <h2>Top Products</h2>
-        <ul class="product-list">
-          <li class="product-card">
-            <a href="product_pages/?product=880RR">
-            <img
-              src="images/tents/marmot-ajax-tent-3-person-3-season-in-pale-pumpkin-terracotta~p~880rr_01~320.jpg"
-              alt="Marmot Ajax tent"
-            />
-            <h3 class="card__brand">Marmot</h3>
-            <h2 class="card__name">Ajax Tent - 3-Person, 3-Season</h2>
-            <p class="product-card__price">$199.99</p></a>
-          </li>
-          <li class="product-card">
-            <a href="product_pages/?product=985RF">
-            <img
-              src="images/tents/the-north-face-talus-tent-4-person-3-season-in-golden-oak-saffron-yellow~p~985rf_01~320.jpg"
-              alt="Talus Tent - 4-Person, 3-Season"
-            />
-            <h3 class="card__brand">The North Face</h3>
-            <h2 class="card__name">Talus Tent - 4-Person, 3-Season</h2>
-            <p class="product-card__price">$199.99</p></a>
-            </li>
-          </li>
-          <li class="product-card">
-            <a href="product_pages/?product=989CG">
-            <img
-              src="images/tents/the-north-face-alpine-guide-tent-3-person-4-season-in-canary-yellow-high-rise-grey~p~985pr_01~320.jpg"
-              alt="Alpine Guide Tent - 3-Person, 4-Season"
-            />
-            <h3 class="card__brand">The North Face</h3>
-            <h2 class="card__name">
-              Alpine Guide Tent - 3-Person, 4-Season
-            </h2>
-            <p class="product-card__price">$349.99</p></a>
-          </li>
-          <li class="product-card">
-            <a href="product_pages/?product=985PR">
-            <img
-              src="images/tents/cedar-ridge-rimrock-tent-2-person-3-season-in-rust-clay~p~344yj_01~320.jpg"
-              alt="Rimrock Tent - 2-Person, 3-Season"
-            />
-            <h3 class="card__brand">Cedar Ridge</h3>
-            <h2 class="card__name">
-              Rimrock Tent - 2-Person, 3-Season
-            </h2>
-            <p class="product-card__price">$69.99</p></a>
-          </li>
-        </ul>
+        <ul class="product-list"></ul>
       </section>
     </main>
-    <footer>&copy;NOT a real business</footer>
+    <footer id="main-footer"></footer>
```

Comment:
Header, footer, and products were made dynamic in `main.js`. This likely caused some confusion. Without the proper `id` attributes the JS code wasn't able to do its thing.